### PR TITLE
Platui 599 integrate contact frontend

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala [hunk 1]
```diff
@@ -23,7 +23,9 @@ import scala.io.Source
 
 @Singleton
 case class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig) {
-  val contactHmrcUnauthenticatedLink = s"${servicesConfig.baseUrl("contact-frontend")}/contact-hmrc-unauthenticated"
+
+  private val contactHost = config.get[String]("contact-frontend.host")
+  val contactHmrcUnauthenticatedLink = s"$contactHost/contact/contact-hmrc-unauthenticated"
```
Comment (matthewmascord) [inline app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala:28]: I think this actually needs to be s"$contactHost/contact/accessibility" - like the link on https://www.tax.service.gov.uk/pay/accessibility-statement

File: app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html [hunk 2]
```diff
@@ -20,8 +20,10 @@
 
 @this(layout: Layout)
 
-@(statement: AccessibilityStatement)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(statement: AccessibilityStatement, userAction: Option[String], referrerUrl: Option[String])(
+        implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 @serviceAbsoluteUrl = {https://@statement.serviceDomain@statement.serviceUrl}
+@contactFrontendUrl = {@{appConfig.contactHmrcUnauthenticatedLink + Seq(Some("?service=" + statement.contactFrontendServiceId),userAction.map(ua => "userAction=" + ua), referrerUrl.map(ru => "referrerUrl=" + ru)).flatten.mkString("&")}}
```
Comment (matthewmascord) [inline app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html:26]: I might be tempted to pull this out into a template of its own?

Comment (matthewmascord) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html:26]: Possibly also have a helper function in the package object to construct the URL

File: app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala (hunk 1 above)
Comment (JoPintoPaul) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala]: Good spot, fixed, thank you.

File: app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html (hunk 2 above)
Comment (JoPintoPaul) [reply app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html]: Moved into the package object for models and added unit tets.

**Pull request merged**